### DISCUSSION TITLE
Custom Output with execute_on=CUSTOM

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#20037] [Q&A General] [answered]
Dear MOOSE Community

I wish to carry out comparisons for mesh convergence at different meshed but at a certain timestep for which I realise that one has to use execute_on = 'CUSTOM' in [Outputs] block but I am not sure how to use this functionality to get the desired output.

Many Thanks
Pushkar

**Comment by GiudGiud** [comment, marked as answer]
Hello

Output to where?
CSV & exodus both output every timestep by default, so you wouldnt need to change this to get all the data available. 

Guillaume

**Reply by ppandit95** [reply]
Thanks @GiudGiud  for elaborating on that and as of now I am outputting an exodus file but is it possible to get a output at certain timestep in VTK or any other visual format

**Reply by GiudGiud** [reply]
Exodus can be visualized in paraview.
VTK can be output at every timestep as well.
Creating a CUSTOM execute_on to select a few timesteps is a little complicated, it's really better to just downselect from the `every timestep` output

**Reply by ppandit95** [reply]
thanks @GiudGiud for elaborating on that...

**Reply by GiudGiud** [reply]
at the end of this page https://mooseframework.inl.gov/source/interfaces/SetupInterface.html
are the instructions to create a CUSTOM execute on flag in your application btw
